tests/pandas_/typesets/test_standard_set_sparse.py

```diff
@@ -77,6 +77,7 @@ def test_contains(name, series, type, member):
     "pd_categorical_sparse": Generic,
     # "datetime_sparse": Generic,
 }
+inference_map = cast(Dict[str, Type[VisionsBaseType]], inference_map)
```

Comment:
Above we use type annotations (PEP 526). Would be more consistent to do that here too